apps/full-site-editing/full-site-editing-plugin/common/index.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * File for various functionality which needs to be added to Simple and Atomic
+ * sites. The code in this file is always loaded.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE\Common;
+
+/**
+ * Adds custom classes to the admin body classes.
+ *
+ * @param string $classes Classes for the body element.
+ * @return string
+ */
+function admin_body_classes( $classes ) {
+	global $post;
+	$is_block_editor_screen = ( function_exists( 'get_current_screen' ) && get_current_screen() && get_current_screen()->is_block_editor() );
+	$hide                   = get_theme_mod( 'hide_front_page_title', false );
```

Comment:
I think we should use a more verbose and readable name here:

```suggestion
	$is_page_title_hidden  = ( true === get_theme_mod( 'hide_front_page_title', false ) );
```